presto-hive/src/test/java/io/prestosql/plugin/hive/parquet/TestParquetDecimalScaling.java

```diff
@@ -82,6 +83,8 @@ protected QueryRunner createQueryRunner()
         basePath = getBasePath();
 
         return HiveQueryRunner.builder()
+                // create nation so tpch schema got created
+                .setInitialTables(ImmutableList.of(NATION))
```

Comment:
maybe instead the q runner should create some other schema to be used for tests that do not need tpch?

Reply:
It looks like it requires not that trivial refactor of `HiveQueryRunner`. I will leave as it is for now.